### DISCUSSION TITLE
Request for Assistance with COTS UE Log Packet Analysis

### DISCUSSION THREAD

**Discussion by pricalinat** [srsran/srsRAN_Project#60] [General Help] [answered]
Currently, I am using a Motorola Edge 30 Pro and I would like to learn how to capture and analyze the signals received by my phone.

I have tried using cellular-pro and Network Signal Guru, but the free versions of these tools do not meet my requirements. I am wondering if there is a way to capture the packets in a log-like format(dlf,qmdl...) and then analyze them using Wireshark.

Thank you for your time and assistance.

**Comment by frankist** [comment]
This is a bit outside of the srsRAN project, but you have tools like QXDM. However, this tool is not free. Not sure, if there is any free alternative.

**Comment by ninjab3s** [comment, marked as answer]
To name some free alternatives, you can use scat [1], qcat [2] or mobile insight [3]. But you have to check if they are compatible with your phone model and Android version. But as @frankist said, this is out of the scope of srsRAN Project.

[1] https://github.com/fgsect/scat
[2] https://github.com/fgsect/scat
[3] http://www.mobileinsight.net/
